Card details.md

```diff
@@ -0,0 +1,313 @@
+            cid: 05e230bf-a9cf-4b31-bc54-d3a995f62526
+    + Body
+
+            {
+                "card": {
+                    "holderName": "Anburaja",
+                    "cardType": "visa",
+                    "cardNumber": "5eeBZ++aHquYp2pmYseO9W3r+1zginDfYs8KsFfHVBk=",
+                    "expiryMonth": "05",
+                    "expiryYear": "2020",
+                    "cvv": "123",
+                    "priorityCard": "Y"
+                },
+                "meta": {}
+            }
+
++ Response 200 (application/json)
+            
+            {
+                "status": "SUCCESS",
+                "data": {
+                    "card": {
+                        "id": "00505683065B1EE78EF34A3F0BD0AF4C",
+                        "holderName": "Anburaja",
+                        "cardType": "visa",
+                        "cardNumber": "5eeBZ++aHquYp2pmYseO9W3r+1zginDfYs8KsFfHVBk=",
+                        "expiryMonth": "05",
+                        "expiryYear": "2020",
+                        "cvv": "123",
+                        "priorityCard": "Y"
+                    }
+                },
+                "errors": {}
+            }
+
++ Request 'Card already available'
+    
+    + Headers
+    
+            cid: 05e230bf-a9cf-4b31-bc54-d3a995f62526
+    
+    + Body
+
+            {
+            "card": {
+                "holderName": "Anburaja",
+                "cardType": "visa",
+                "cardNumber": "5eeBZ++aHquYp2pmYseO9W3r+1zginDfYs8KsFfHVBk=",
+                "expiryMonth": "05",
+                "expiryYear": "2020",
+                "cvv": "123",
+                "priorityCard": "Y"
+            },
+            "meta": {}
+            }
+    
++ Response 400 (application/json)
+
+        {
+            "status": "FAIL",
+            "data": {},
+            "errors": [{
+                "code": "card",
+                "message": "error.card.already.available"
+            }],
+            "meta": {}
+        }
+        
++ Request 'Service unavailable to save card Detail'
+    
+        + Headers
+            cid: 05e230bf-a9cf-4b31-bc54-d3a995f62526
+    
+    + Body
+
+            {
+            "card": {
+                "holderName": "Anburaja",
+                "cardType": "visa",
+                "cardNumber": "5eeBZ++aHquYp2pmYseO9W3r+1zginDfYs8KsFfHVBk=",
+                "expiryMonth": "05",
+                "expiryYear": "2020",
+                "cvv": "123",
+                "priorityCard": "Y"
+            },
+            "meta": {}
+            }
+
++ Response 500 (application/json)
+
+        {
+            "status": "FAIL",
+            "data": {},
+            "errors": [{
+                "code": "Service unavailable",
+                "message": "error.sap.pi.operation.failure"
+            }],
+            "meta": {}
+        }
+
+### Edit card Detail [PUT /cards/{id}]
+
++ Parameters
+
+    + id: 00505683065B1EE78EF34A3F0BD0AF4C (string) - SaveCardId
```

Comment:
Is `PUT` applicable only for anonymous customer? If no can we add access token in the headers too?